Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -120,10 +123,11 @@ public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder)
                     $value = $this->getFilterValueFromUrl($value);
                 }
 
-                $valueParameter = QueryUtils::generateParameterName($field);
+                $valueParameter = QueryUtils::generateParameterName($this->isPropertyNested($property) ? $alias.'_'.$field : $field);
```

Comment:
Is this necessary? Because collision shouldn't be a problem here. This would mean extra-long parameter names...